bootstrap/cmd/kfctl/cmd/apply/apply.go

```diff
@@ -0,0 +1,112 @@
+package apply
```

Comment:
At cmd/kfctl/cmd/apply.go will be present and then a new directory cmd/kfctl/cmd/apply/ will contain the bootstrapping. Would this be the format followed for the "new" kfctl? for generate as well?